testing/PythonTests/mapping_navigation_test.py

```diff
@@ -0,0 +1,10 @@
+from pure_pursuit import CarState
+from map_information import Point
+
+
+class TestPurePursuit:
+    def test_car_state_update_pos(self):
+        state = CarState(5)
+        state.update_pos(Point((2, 2)), (4, 6, 8, 10))
+        print(f'{state.rear_x}, {state.rear_y}, {state.yaw}')
+        assert True
```

Comment:
What is assert true doing here? There's no condition it's checking so won't it always be true?

Reply:
Yep, this is a stub test for now so I could test with multiple functions. I will flesh it out in another PR later this week.

Reply:
Ah kk